### PR TITLE
JS Capstone: Popup

### PR DESCRIPTION
**Pull request by Kwaku28**
## This part of the project is configure the popup window

Functions added include the following:

1. A function to display the window
2. A function to display details of an item when clicked
3. A function to display the comment section
4. A function to include the form to post comments

Thank you

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { updateLikes } from './module/aboutLikes.js';
 
 displayList();
 addHeaderToPage();
+updateLikes();
+// countItems();
```
Comment (KUBAHO3) [inline src/index.js]: Please consider not committing the comments because it makes your code look messy

**Review by KUBAHO3 — COMMENTED** [review]
image
Well done so far! 

> The Task functionality is working more fine But you need to check the changes in the comment section in order to make your code look even better

File: src/index.js (hunk 1 above)
Comment (Kwaku28) [reply src/index.js]: As suggested, I have removed the comments from the code. Thank you

**Comment by KUBAHO3** [comment]
Awesome! Thank you for considering making changes on the mentioned slice of code.
Now your project is **Approved!**
approved

**Pull request merged**